lib/rspec/mocks/message_expectation.rb

```diff
@@ -563,10 +569,6 @@ def raise_already_invoked_error_if_necessary(calling_customization)
           error_generator.raise_already_invoked_error(message, calling_customization)
         end
 
-        def failed_fast?
-          @failed_fast
-        end
```

Comment:
I realized that it is no longer used.  It used to be used but the usage was removed in #884.  `failed_fast?` and `@failed_fast` were vestiges of old logic that we no longer have a reason to keep around.